lib/ui/view/channel/channel_view/widgets/channel_chat.dart

```diff
@@ -27,26 +29,55 @@ class ChannelChat extends ViewModelWidget<ChannelPageViewModel> {
               itemBuilder: (context, index) {
                 final userPost = viewModel.channelUserMessages![index];
                 return InkWell(
-                  child: ThreadCardView.threadChannelMain(userPost),
+                  child: Container(
+                    padding: const EdgeInsets.only(top: 20.0),
+                    child: Column(
+                      children: [
+                        if (userPost.pinned) ...[
+                          Padding(
+                            padding: const EdgeInsets.only(left: 40.0),
+                            child: Row(
+                              children: [
+                                const Icon(Icons.push_pin,
+                                    size: 16.0, color: Colors.orange),
+                                const SizedBox(width: 12.0),
+                                Text(Pinned,
+                                    style: GoogleFonts.lato(
```

Comment:
Please look for the style in apptextstyles.dart and if there is none, create one but check first